rbac/management/group/view.py

```diff
@@ -482,14 +483,26 @@ def roles(self, request, uuid=None):
 
         return Response(status=status.HTTP_200_OK, data=response_data.data)
 
+    def filtered_roles(self, roles, request):
+        """Return filtered roles for group from query params."""
+        role_filters = {}
+        for param_name, param_value in request.query_params.items():
+            if param_name in VALID_GROUP_ROLE_FILTERS:
+                attr_filter_name = param_name.replace('role_', '')
```

Comment:
Could we just use 'name' and 'description' without the 'role_'? which is easier to type

Reply:
@astrozzc unfortunately, it doesn't appear so. I noted in the description of the PR the issue I was running into with the `name` parameter. Might be worth looking into further, though.